### ISSUE TITLE
How can I get lat & lon coordinates from an image?

### ISSUE THREAD

**Issue by xlcrr** [NativeScript/nativescript-imagepicker#183]
How can I get lat & lon coordinates from the exif file from an image?

**Comment by lini** [comment]
If you are using Android, you can try the [Media Metadata Retriever](https://market.nativescript.org/plugins/nativescript-media-metadata-retriever) plugin, or if you want to access the native API directly - [android.media.ExifInterface](https://developer.android.com/reference/android/media/ExifInterface.html#getLatLong(float[]))

For iOS, you should be able to call the native API with the PHAsset that is returned from the image picker plugin. Example (where `selection` is the returned array from the image picker plugin):

```javascript
const ios = selection[0].ios;
console.log(ios.location);
```

**Comment by lini** [comment]
Closing due to inactivity.

**Comment by Spacarar** [comment]
I know this is kind of old and closed, but I as well as some of us over on the Nativescript-Vue slack channel have been struggling to retrieve this metadata out of the image. As far as I have tested on my ios device: putting `selection[0].ios` as a variable in the store I see that it is an `PHAsset` but if you log it, it shows as an empty object. Using `selection[0].ios.location` (even as two lines as you showed) returns `undefined`.  I know the photos I have chosen have additional data as my phone will show me when/where they are taken.

As far as I can tell in all the logging I have done each image comes across as an object with
```{
  _observers: {},
  _options: {
    keepAspectRatio: true,
    autoScaleFactor: true
  },
  _ios: {}
}
```
Everything I have tried to get the metadata out of the photo has been a total bust returning either undefined, null or {}
